bobgit/add/add_target.go

```diff
@@ -0,0 +1,134 @@
+package add
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Benchkram/bob/pkg/filepathutil"
+)
+
+type RepoTargetMap map[string]string
```

Comment:
Comment. What are the key/values of this map

Reply:
coment added